src/components/Item/item.js

```diff
@@ -24,27 +26,22 @@ export default function Item () {
   const navLocation = useLocation()
 
   const { itemKey, gender } = useParams()
-  const currency = useSelector(state => state.currency)
-  const size = useSelector(state => state.size)
-
-  useAPICall('getitem', {
-    itemKey: decodeURIComponent(itemKey),
-    gender,
-    size,
-    fromBrowse: navLocation.state
-  })
 
+  const user = useSelector(state => state.user)
+  const size = useSelector(state => state.size)
+  const currency = useSelector(state => state.currency)
   const itemInfo = useSelector(state => state.item.itemInfo)
   const itemPrices = useSelector(state => state.item.itemPrices)
   const itemListings = useSelector(state => state.item.itemListings)
   const relatedItems = useSelector(state => state.item.relatedItems)
-
   const pricesLoading = useSelector(state => state.item.loadingItemPrices)
```

Comment:
we should really stop abusing global state lol

Reply:
idk all these states are pretty necessary b/c each one corresponds to a blocking api call. we wouldn't want one big state that we have to wait for until everything has loaded - would just slow things down a lot.